docs/release-notes/august-2023.md

```diff
@@ -0,0 +1,176 @@
+```
+map($Y) do $X
+  ...
+end
+```
+- **Java:** Fixed a naming bug affecting Java and other object-oriented (OO) languages that allowed a method parameter to shadow a class attribute. For example, in:
+  ```
+  class Test {
+  
+      private int x;
+  
+      public void test2(int x) {
+          foo(this.x);
+      }
+  
+  }
+  ```
+  Semgrep was considering that `this.x` referred to the parameter `x` of `test2` rather than to the class attribute `x`. ([#8508](https://github.com/returntocorp/semgrep/pull/8508))
+
+
+### Changed
+
+- Running the `semgrep`  command with no subcommand now displays the help message. Previously, the `semgrep` command ran a SAST scan by default.
+
+### Removed
+
+- `python -m semgrep` has been removed. Instead, invoke Semgrep directly by entering `semgrep` in the CLI.
+- Semgrep no longer looks for a `.semgrep.yml` config file or `.semgrep/` in the current directory, which previously caused conflicts when invoking `semgrep` from your home directory. This is because the home directory can contain a `.semgrep/settings.yml` file that is not a Semgrep rule, despite Semgrep expecting a rule file. ([#4457](https://github.com/returntocorp/semgrep/issues/4457))
+  - The preferred method to run rules is to explicitly pass rules through the `--config` option. For example, to run a `.semgrep.yml` file containing rules, you must enter `semgrep --config .semgrep.yml`.
+- If you previously wrapped Semgrep Python code by calling `semgrep_main.main`, you must replace the previous call with `run_scan.run_scan`. Note that these Python calls will be removed in the future.
+- `--enable-metrics` and `--disable-metrics` have been removed. Instead, use any of the following:
+  - `--metrics=on`
+  - `--metrics=off`
+  - `--metrics=auto`
+
+## Semgrep Pro Engine
+
+<!-- Separated pro engine because it is proprietary -->
+
+### Fixed
+
+- **JavaScipt (JS) or TypeScript (TS) taint mode:** fixed a bug introduced in 1.33.1 that had the side-effect of hurting performance of taint rules on JS or TS repositories that used destructuring in a function's formal parameters.
+
+## Semgrep Cloud Platform
+
+### Added
+
+- The `semgrep ci` command now displays enabled products when scans are command now created or when the scan config is generated from Semgrep Cloud Platform. Additionally, if no products are enabled then a friendly error is raised and the scan is stopped. You must enable a product in Semgrep **Cloud Platform > Settings** to start a scan.
+- You can now remove your SSO configuration. Previously, you had to reach out to [support@semgrep.com](mailto:support@semgrep.com) to remove SSO configurations.
+- **Projects page:** Added a **Sync projects** button which enables you to synchronize your Semgrep projects with your SCM. This means you are able to see the current status of your scans across all your projects that you have onboarded to Semgrep Cloud Platform.
```

Comment:
"This means you are able to see the current status of your scans across all your projects that you have onboarded to Semgrep Cloud Platform." -> @ThatDevShparki doesn't this technically just mean that the projects will be there and it's more for enabling easier onboarding of additional projects? Being able to see the status was already there?

Reply:
That's correct- the sync button just allows you to manually synchronize your current repository list with those available in the SCMs where those repositories originated from

Reply:
@s-santillan I think we should rephrase this to say "Added a **Sync projects** button which enables you to synchronize your Semgrep projects with your SCM. This will enable you to faster onboard projects to the Semgrep Cloud Platform and ensure all your repositories are represented and available for scanning." (or something similar)

Reply:
Sounds perfect. Thank you @annikalee and @ThatDevShparki

Reply:
Resolving as fixed in latest commit.